src/App.scss

```diff
@@ -8,13 +8,17 @@
 
 .pf-c-content {
     .ins-m-tech-preview {
-        color: var(--pf-global--success-color--100) !important;
+        color: var(--pf-global--success-color--200) !important;
```

Comment:
We really should try to avoid using `!important`. Just make the selectors more specific.